ddtrace/tracer/spancontext.go

```diff
@@ -93,17 +93,13 @@ func (c *spanContext) setSamplingPriority(p int) {
 	c.trace.setSamplingPriority(float64(p))
 }
 
-func (c *spanContext) samplingPriority() int {
+func (c *spanContext) samplingPriority() (int, bool) {
```

Comment:
For the sake of documentation
```suggestion
func (c *spanContext) samplingPriority() (p int, ok bool) {
```